source/loader/windows/loader_location.cpp

```diff
@@ -0,0 +1,42 @@
+/*
+ *
+ * Copyright (C) 2023 Intel Corporation
+ *
+ * SPDX-License-Identifier: MIT
+ *
+ */
+
+/* This include needs to be before Libloaderapi.h */
+#include <windows.h>
+
+#include <Libloaderapi.h>
+
+#if __has_include(<filesystem>)
+#include <filesystem>
+namespace filesystem = std::filesystem;
+#else
+#include <experimental/filesystem>
+namespace filesystem = std::experimental::filesystem;
+#endif #include < filesystem>
+
+#include "ur_loader.hpp"
+
+#define MAX_PATH_LEN_WIN 1024
```

Comment:
why 1024? MAX_PATH is 256 (default) or 32K (need changes in registry)

Reply:
That was an arbitrary number that I was going to change in the future but forgot.
I've made the buffer bigger, equal to the maximum possible path length according to https://learn.microsoft.com/en-us/windows/win32/fileio/maximum-file-path-limitation?tabs=registry.